### PR TITLE
Bug 2176727: add ip address under ipv4 if the user has no dhcp

### PR DESCRIPTION
**Pull request by upalatucci**
if the user chooses IPV4 but does not want DHCP, he should be able to set an IP address.

Screenshot from 2023-03-15 14-38-33
Screenshot from 2023-03-15 14-30-55
Screenshot from 2023-03-15 14-31-00

### CODE REVIEW

File: locales/en/plugin__nmstate-console-plugin.json [hunk 1]
```diff
@@ -49,9 +49,11 @@
   "Interface name": "Interface name",
   "Interface name should follow the linux kernel naming convention. The name should be smaller than 16 characters.": "Interface name should follow the linux kernel naming convention. The name should be smaller than 16 characters.",
   "Interface name should follow the linux kernel naming convention. Whitespaces and slashes are not allowed.": "Interface name should follow the linux kernel naming convention. Whitespaces and slashes are not allowed.",
+  "IP address": "IP address",
   "IP configuration": "IP configuration",
   "IPv4": "IPv4",
   "IPV4": "IPV4",
+  "IPV4 address": "IPV4 address",
```
Comment (pcbailey) [inline locales/en/plugin__nmstate-console-plugin.json:55]: It's typically written 'IPv4' with a lowercase 'v'.

File: src/utils/components/PolicyForm/PolicyInterface.tsx [hunk 2]
```diff
@@ -195,17 +199,42 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
         <Checkbox
           label={t('IPV4')}
           id={`policy-interface-ip-${id}`}
-          isChecked={!!policyInterface.ipv4}
+          isChecked={policyInterface.ipv4?.enabled}
```
Comment (pcbailey) [inline src/utils/components/PolicyForm/PolicyInterface.tsx]: Is `ipv4` guaranteed to be there? If not, could you please add optional chaining there and the other places it's used?

**Review by pcbailey — APPROVED** [review]

**Comment by jupierce** [comment]
CNV QE reached out to me asking why this PR was not included in the most recent 4.14 builds being performed by ART. Please note that this PR merged into `main` and not the branch that ART builds 4.14 content from: `release-4.14` . If the intent was for this PR to affect 4.14 builds, PRs will need to be cherry-picked to 4.14 or automated fast-forwarding will need to be setup.

**Pull request merged**